src/nvim/digraph.c

```diff
@@ -1833,27 +1833,33 @@ char_u* keymap_init(void)
   } else {
     char *buf;
     size_t buflen;
+    int result = FAIL;
+    unsigned int cnt = 0;  // Counter for going through fmts
+
+    char *fmts[] = {
+      "keymap/%s_%s.vim",
+      "keymap/%s_%s.lua",
+      "keymap/%s.vim",
+      "keymap/%s.lua",
+    };
```

Comment:
So I'm not sure that keymap makes sense for lua (haven't read all the comments yet) but it doesn't really execute `vimscript` in keymap files, right?

Reply:
Well I don't think keymap makes sense either . Added because @clason said to :P

Reply:
That's right, blame me :P 

But you've actually already convinced me that `keymap` doesn't make sense here (no vim => no Lua)

Reply:
So we agree on removing keymap ?

Reply:
Yes. Also `autoload` makes little sense since the `foo#bar` syntax isn't easy to support -- we can punt to `/lua` and `require('foo').bar()` for that.

Reply:
Ok. I've reverted the keymap commit.

Also @clason `autoload` is in this pr . It's very vim specific . So I agree it can't be lua :)